packages/reactive-element/src/test/reactive-element_test.ts

```diff
@@ -2781,157 +2781,6 @@ suite('ReactiveElement', () => {
     });
   });
 
```

Comment:
Add a test to verify that updates happen while disconnected?

Reply:
Done